buildtools/sample-server/main.js

```diff
@@ -0,0 +1,73 @@
+/* global OktaSignIn */
+/* eslint no-console: 0 */
+
+import widgetRc from '../../.widgetrc';
```

Comment:
Just curious - this is automatically resolved to using the json loader?

Reply:
yeah, basically.
```
{
        test: /\.json$|\.widgetrc/,
        loader: 'json-loader'
      },
```

Reply:
Hah! I checked the wrong config file - common.config rather than playground.config